pkg/operator/ceph/config/datapath.go

```diff
@@ -39,6 +39,15 @@ type DataPathMap struct {
 	// persisted to the host. The log dir is always /var/log/ceph. If logs are not persisted to the
 	// host, logs are not shared between containers via empty dir or any other mechanism.
 	HostLogDir string
+
+	// ContainerLogDir represents Ceph's logging directory
+	ContainerLogDir string
```

Comment:
The log dir is constant in the container. I would  suggest removing this so that it doesn't invite misuse in the future. This value existed before and was removed for precisely this reason. 

I have now seen you add in a constant value to this same mutable struct twice now. I am sensing that having this parameter seems useful to you, and I would agree. One would want to ask the data path map for the in-container dirs as well. That is more straightforward than remembering which constant to use. But instead of relying on future devs to always properly set this value, what if we create a function on the data path map that always returns the constant value?

```go
func (d *DataPathMap) ContainerLogDir() string {
	return config.VarLogCephDir
}
```